src/platform/Tizen/ConnectivityUtils.cpp

```diff
@@ -15,16 +15,11 @@
  *    limitations under the License.
  */
 
-#include "ConnectivityUtils.h"
```

Comment:
Same as above. Keeping this include at the top, validates the include itself.

Reply:
I'll update it.